just_audio_background/lib/just_audio_background.dart

```diff
@@ -166,7 +166,15 @@ class _JustAudioPlayer extends AudioPlayerPlatform {
   late final _PlayerAudioHandler _playerAudioHandler;
 
   _JustAudioPlayer({required this.initRequest}) : super(initRequest.id) {
-    _playerAudioHandler = _PlayerAudioHandler(initRequest);
+    _playerAudioHandler = _PlayerAudioHandler(
+      initRequest,
+      addPlaybackEventMessageError: (error, stackTrace) {
+        if (eventController.isClosed) {
```

Comment:
This workaround will not work. 
eventController already closed, because disposePlayer called and after that player?.release();

```
Future<void> release() async {
    eventController.close();
    await _audioHandler.stop();
  }
```